src/base_config.js

```diff
@@ -180,7 +181,8 @@ class BaseConfig {
           })
           .catch((response) => {
             const thunk = this._genericFailure(type);
-            const errorsObject = formatErrorResponse(response);
+            const errorsObject =
```

Comment:
I'm curious why you chose to do it this way versus using a ternary? Does this work the same way? Another option that might make this line easier to read would be to set a default function in the constructor which would allow this line to be:

```js
const errorsObject = this.parseServerErrorsFunc(response);
```

The constructor would then look like:

```js
constructor(inputs) {
  ...
  this.parseServerErrorsFunc = inputs.parseServerErrorsFunc || (errors) => errors;
  ...
}

Reply:
Not sure why I chose to not use a ternary. I think  that works better.